### ISSUE TITLE
Variables not found for dependencies noted in local `Project.toml`.

### ISSUE THREAD

**Issue by lassepe** [JuliaEditorSupport/LanguageServer.jl#349]
I usually have all my dependencies noted in a project specific `Project.toml` in an effort to keep the number of dependencies installed globally (e.g. noted in `~/.julia/environments/Project.toml`) small.

However, if I open a file belonging to a project with a local `Project.toml` in my editor, all variables that are declared by packages that are not installed globally are marked as
*"Missing variable. Use of possibly undeclared Variable"*.

I launch my language server from within vim as described in the wiki. That is, I run the `julia` binary with flags `--startup-file=no`, `--history-file=no` and 
```
-e 'using LanguageServer; using Pkg; import StaticLint; import SymbolServer; env_path = dirname(Pkg.Types.Context().env.project_file); debug = false;  server = LanguageServer.LanguageServerInstance(stdin, stdout, debug, env_path, "", Dict()); server.runlinter = true; run(server);'
```
However, this seems to always load the `Project.toml` from within `~/.julia/environments/` and ignores the local `Project.toml`.

**TLDR:** Is there a way to have the language server know about the dependencies declared in the local `Project.toml`?

**Comment by davidanthoff** [comment]
The currently shipping versions doesn't handle this case properly, in fact it _always_ just resolves names in the global default environment.

https://github.com/JuliaEditorSupport/LanguageServer.jl/pull/347/files should fix this, but it doesn't yet fully work. It also only works well with the latest `master` version of the VS Code extension. So, long story short: I think your best option is to wait until we finish the next version of the extension, and then this will be resolved, hopefully.

**Comment by davidanthoff** [comment]
I'm going to close this because we are tracking support for julia env in the PR already.

**Comment by lassepe** [comment]
Thank you!
